content/en/docs/setup/install/helm/index.md

```diff
@@ -57,13 +57,6 @@ $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{
 Change directory to the root of the release and then
 choose one of the following two **mutually exclusive** options:
 
-{{< warning >}}
```

Comment:
I agree we should remove this from here, but we should add an "Installation Problems" entry in https://preliminary.istio.io/docs/ops/troubleshooting/ that explains this problem.

Reply:
I disagree. I have never heard of anyone running into this problem except the person who added the note here. We shouldn't document every single problem someone could possibly run into.

To me this is equivalent to saying "if you run `hlem` instead of `helm` the install won't work, so don't make a typo.

If you disagree that's fine though.. not a docs expert

Reply:
Fair enough. If Steve is the one and only person to have ever had this problem, I agree we don't need to document it. We should only document commonly occuring problems.